imports/plugins/core/accounts/server/methods/userAccountQuery.js

```diff
@@ -0,0 +1,29 @@
+import { Meteor } from "meteor/meteor";
+import { Accounts } from "/lib/collections";
+import { Reaction } from "/lib/api";
+
+/**
+ * @name userAccountQuery
+ * @method
+ * @summary query the Accounts collection and return user account data
+ * @param {Object} context - an object containing the per-request state
+ * @param {String} id - id of user to query
+ * @return {Object} user account object
+ */
+export function userAccountQuery(context, id) {
+  const userId = context.user._id;
+
+  // Check to make sure current user has permissions to view queried user
+  if (!Reaction.hasPermission("reaction-accounts", userId)) throw new Meteor.Error("User does not have permission");
```

Comment:
For the errors, follow https://docs.reactioncommerce.com/reaction-docs/master/error-handling-guide

Specifically, make sure there's a machine-readable designator as the first argument, since we'll eventually need to evaluate errors we receive in the browser code, and decide what kind of UI message to show.

example for this one: `throw new Meteor.Error("access-denied", "User does not have permission")`